google/async_resumable_media/_download.py

```diff
@@ -124,13 +125,6 @@ class Download(DownloadBase):
             ``start`` to the end of the media.
         headers (Optional[Mapping[str, str]]): Extra headers that should
             be sent with the request, e.g. headers for encrypted data.
-        checksum Optional([str]): The type of checksum to compute to verify
-            the integrity of the object. The response headers must contain
-            a checksum of the requested type. If the headers lack an
```

Comment:
Looks like you re-added checksum (good!) but you still are missing this docstring piece-- add this back in.

Reply:
Added this back in.